python3.11/apigw-pytorch/{{cookiecutter.project_name}}/app/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM public.ecr.aws/lambda/python:3.11-preview
```

Comment:
Is this still in preview or can we remove it?

Reply:
good catch. I updated it in the build images and missed it here.

Reply:
resolved